.github/workflows/tests-rs-package.yml

```diff
@@ -151,3 +211,53 @@ jobs:
           SCCACHE_BUCKET: multi-runner-cache-x1xibo9c
           SCCACHE_REGION: ${{ vars.AWS_REGION }}
           SCCACHE_S3_KEY_PREFIX: ${{ runner.os }}/sccache/${{ steps.arch.outputs.arch }}/linux-gnu
+
+  check_each_feature:
+    name: Check each feature
+    runs-on: ${{ fromJSON(inputs.test-runner) }}
+    needs:
```

Comment:
I don't think it's a good idea. It will increase overall time for couple minutes (just to wait and bootstrap runner). It will be cheaper just to read cargo twice.

Reply:
refactored - moved crate_info to .github/actions